examples/protobuf/Thermostat/Client/Program.cs

```diff
@@ -28,38 +28,42 @@
 // set <value> changes the set point and exits.
 
 var monitorCommand = new Command("monitor", "Monitor the thermostat (default)");
-monitorCommand.SetHandler(MonitorAsync);
+monitorCommand.SetAction((parseResult, cancellationToken) => MonitorAsync(cancellationToken));
```

Comment:
I meant, keep `monitorCommand.SetAction(MonitorAsync)` and change the signature of `MonitorAsync`.